3.10/deployment-kubernetes-deployment-resource.md

```diff
@@ -62,6 +62,55 @@ with `<group>` where `<group>` can be any of:
 - `syncmasters` for all syncmasters of a `Cluster`.
 - `syncworkers` for all syncworkers of a `Cluster`.
 
+### `spec.architectures: []string`
+
+This setting specifies a list of CPU architectures for the deployment.
+Possible values are:
+
+- `amd64` (default): Use processors with the x86-64 architecture.
+- `arm64`: Use processors with the 64-bit ARM architecture.
+
+To let new members as well as recreated members use `arm64` nodes, modify the
```

Comment:
I guess we should link/mention here that this has to be enabled in the operator first (as described above)